sdk/rust/src/ledger_id.rs

```diff
@@ -45,6 +55,10 @@ impl LedgerId {
     pub fn is_known_network(&self) -> bool {
         self.is_mainnet() || self.is_previewnet() || self.is_testnet()
     }
+
+    pub const fn to_bytes(&self) -> &Vec<u8> {
```

Comment:
that would be `as_bytes` with the current signature (https://rust-lang.github.io/api-guidelines/naming.html#ad-hoc-conversions-follow-as_-to_-into_-conventions-c-conv)... and technically you'd want to return a ` &[u8]` (there's basically nothing you can do with a `&Vec<T>` that you can't do with a `&[u8]`)

anyway, for `to_bytes` you'd want to return a `Vec<u8>` (`self.0.clone()`)

Reply:
`to_bytes()` is specified in the SDK spec.  Should we have both `as_bytes()` and `to_bytes()`?  Is there any reason to have the `to_bytes()` implementation you suggested here?  Couldn't the user simply `clone()` it themselves if they want?